workshops/models.py

```diff
@@ -369,6 +364,14 @@ def save(self, *args, **kwargs):
         super().save(*args, **kwargs)
 
 
+def is_admin(user):
+    if user is None or user.is_anonymous():
+        return False
+    else:
+        is_admin = user.groups.filter(name='administrators').exists()
+        return is_admin or user.is_staff or user.is_superuser
```

Comment:
1. I'm unsure the use of `user.is_staff`. What if we change its meaning later on so that it will break logic here? Dropping `or user.is_staff` condition here would remove the only use of `user.is_staff` we have right now -- so this field is only used by the Django auth.

2. I'd rename `is_admin` variable from inside `is_admin` function to something like `has_admin_group`.